.github/workflows/lint.yaml

```diff
@@ -0,0 +1,12 @@
+on: push
+name: Clippy check
+jobs:
+  clippy_check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - run: rustup component add clippy
```

Comment:
does this action not need the toolchain action from above? Do they end up with the same version?